src/IonDecimal.ts

```diff
@@ -49,27 +49,68 @@ export class Decimal {
     public static readonly ONE = new Decimal(1, 0);
 
     /**
-     * Creates a new Decimal value.
-     * @param coefficient   See the {@link Decimal} class-level documentation for details.
-     * @param exponent      See the {@link Decimal} class-level documentation for details.
+     * Creates a new Decimal value. Three parameter sets are supported:
+     *
+     *  (text: string)
+     *      The provided text will be parsed into a Decimal value.
+     *
+     *  (coefficient: number, coefficient: number)
+     *      A Decimal value will be constructed using the provided coefficient and exponent values.
+     *
+     *  (coefficient: BigInt, coefficient: number, isNegative?: boolean)
+     *      A Decimal value will be constructed using the provided coefficient and exponent values.
+     *      Because the BigInt data type cannot represent -0, a third parameter called `isNegative` may be supplied
+     *      to explicitly set the sign of the Decimal following construction. If this flag is not specified,
+     *      the provided coefficient's sign will be used.
```

Comment:
Technically aren't both variants `number | JSBI` for the coefficient?

What is the implication of a negative coefficient and `isNegative` being in conflict or not with that?  Is the sign of the coefficient ignored--that seems to be the implication, but I would be explicit.

Reply:
An unfortunate consequence of Javascript/Typescript not supporting constructor overloading is that all of the parameter sets get intermingled.

> Technically aren't both variants number | JSBI for the coefficient?

I would say they're not because the code path for `(number, number)` will ignore a third parameter (`isNegative`) while the code path for `(BigInt, number, isNegative?)` will respect `isNegative` when it's supplied.

> What is the implication of a negative coefficient and isNegative being in conflict or not with that? Is the sign of the coefficient ignored--that seems to be the implication, but I would be explicit.

Good point. The sign of the coefficient will be overridden by `isNegative` if specified. I'll add that to the documentation, thanks.